static/templates/rt_resource_template_doc.json

```diff
@@ -154,6 +154,35 @@
       }
     ]
   },
+  {
+    "@id": "_:b6",
+    "@type": ["http://sinopia.io/vocabulary/PropertyTemplate"],
+    "http://www.w3.org/2000/01/rdf-schema#label": [
+      {
+        "@value": "Optional classes"
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasRemark": [
+      {
+        "@value": "Optional classes for the resource."
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasPropertyUri": [
+      {
+        "@id": "http://sinopia.io/vocabulary/hasOptionalClass"
```

Comment:
I suspect I missed it ... but is there a test that exercises this?  Do we update a fixture with this or otherwise exercise this property?   I saw plenty with multiple classes, so yes, implied ... but ??

Reply:
Yes, the optional classes are what allows multiple classes.